src/js/cookies-settings.js

```diff
@@ -55,31 +60,33 @@ export default class CookiesSettings {
     }
     setConsentCookie(options);
 
-    this.checkForPreviousPage();
     this.showConfirmationMessage();
     this.hideCookiesBanner();
 
     return false;
   }
 
-  checkForPreviousPage() {
-    if (1 < history.length) {
-      event.preventDefault();
-      window.history.back();
-    }
-  }
-
   showConfirmationMessage() {
-    const confirmationMessage = document.querySelector('.cookies-confirmation-message');
-
-    document.body.scrollTop = document.documentElement.scrollTop = 0;
-    confirmationMessage.classList.remove('u-d-no');
+    this.confirmationMessage.classList.remove('u-d-no');
+    this.confirmationMessage.scrollIntoView();
```

Comment:
I think scrolling to top of body was better, as it mimics a page reload and consistent with error validation.  It does look disorientating without the context of the page header in view.